pandas/core/generic.py

```diff
@@ -4091,8 +4090,7 @@ def sort_values(self, by=None, axis=0, ascending=True, inplace=False,
         0   A    2    0
         1   A    1    1
         """
-        raise NotImplementedError("sort_values has not been implemented "
-                                  "on Panel or Panel4D objects.")
+        raise NotImplementedError("sort_values has not been implemented.")
```

Comment:
Since this is now never hit, should this be AbstractMethodError?

Reply:
Yea I think could go for that too